### PR TITLE
Enhancement: Account center updates

### PR DESCRIPTION
**Pull request by leightkt**
### Description
Update Account Center default location, animation, and position of components.

### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [x] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### Docs Checklist
- [x] Include a screenshot of any changes ([see docs README on running locally](https://github.com/blocknative/web3-onboard/blob/develop/docs/README.md))
- [x] Add/update the appropriate package README (if applicable)
- [x] Add/update the related module in the [docs demo](https://github.com/blocknative/web3-onboard/blob/develop/docs/src/lib/services/onboard.js) (if applicable)
- [x] Add/update the related package in the `docs/package.json` file (if applicable)

Screenshot 2023-05-24 at 4 29 50 PM

### CODE REVIEW

**Review by gesquinca — APPROVED** [review]

File: docs/src/routes/docs/[...3]modules/[...1]core/+page.md [hunk 1]
```diff
@@ -17,6 +17,22 @@ title: Core
 
 This is the core package that contains all of the UI and logic to be able to seamlessly connect user's wallets to your app and track the state of those wallets. Onboard no longer contains any wallet specific code, so wallets need to be passed in upon initialization.
 
+:::admonition type="tip"
+_note: Release 2.23.0 moves the default position of the account center from topRight to bottomRight. To reset your application to topRight, include the following when initializing onboard:_
```
Comment (Adamj1232) [inline docs/src/routes/docs/[...3]modules/[...1]core/+page.md:21]:
```suggestion
_note: Release 2.24.0 moves the default position of the account center from topRight to bottomRight. To reset your application to topRight, include the following when initializing onboard:_
```

File: packages/core/README.md [hunk 2]
```diff
@@ -6,6 +6,20 @@
 
 This is the core package that contains all of the UI and logic to be able to seamlessly connect user's wallets to your app and track the state of those wallets. Onboard no longer contains any wallet specific code, so wallets need to be passed in upon initialization.
 
+_Tip: Release 2.23.0 moves the default position of the account center from topRight to bottomRight. To reset your application to topRight, include the following when initializing onboard:_
```
Comment (Adamj1232) [inline packages/core/README.md:9]:
```suggestion
_Tip: Release 2.24.0 moves the default position of the account center from topRight to bottomRight. To reset your application to topRight, include the following when initializing onboard:_
```

File: packages/core/src/views/account-center/AcctCenterTriggerSmall.svelte [hunk 3]
```diff
@@ -1,49 +1,56 @@
...
-  .drop-shadow {
-    filter: drop-shadow(0px 1px 4px rgba(0, 0, 0, 0.2));
-  }
-
-  .inner-row {
-    display: flex;
-    flex-flow: row nowrap;
-    width: 80px;
-    padding: 0.75rem;
+    z-index: 1;
```
Comment (Adamj1232) [inline packages/core/src/views/account-center/AcctCenterTriggerSmall.svelte:53]: Same as above z-index note

**Comment by Adamj1232** [comment]
Also getting some duplicates within the react demo when linking and changing the position of account center with the Max AC opened
Screenshot 2023-05-25 at 8 58 56 AM

**Review by Adamj1232 — APPROVED** [review]
Nice work!!

**Pull request merged**